docs/contributing/codebase-overview.md

```diff
@@ -65,40 +66,45 @@ The source code is located in `/api/src` and the below folders are inside there.
+| [@directus/storage-driver-azure](https://github.com/directus/directus/tree/main/packages/storage-driver-azure)           | Azure file storage abstraction for `@directus/storage`                                                                           |
+| [@directus/storage-driver-cloudinary](https://github.com/directus/directus/tree/main/packages/storage-driver-cloudinary) | Cloudinary file storage abstraction for `@directus/storage`                                                                      |
+| [@directus/storage-driver-gcs](https://github.com/directus/directus/tree/main/packages/storage-driver-gcs)               | GCS file storage abstraction for `@directus/storage`                                                                             |
+| [@directus/storage-driver-local](https://github.com/directus/directus/tree/main/packages/storage-driver-local)           | Local file storage abstraction for `@directus/storage`                                                                           |
+| [@directus/storage-driver-s3](https://github.com/directus/directus/tree/main/packages/storage-driver-s3)                 | S3 file storage abstraction for `@directus/storage`                                                                              |
+| [@directus/storage](https://github.com/directus/directus/tree/main/packages/storage)                                     | Object storage abstraction layer for Directus                                                                                    |
+| [@directus/tsconfig](https://github.com/directus/directus/tree/main/packages/tsconfig)                                   | The shared TS Config files used by the projects in the Directus ecosystem.                                                       |
+| [@directus/types](https://github.com/directus/directus/tree/main/packages/types)                                         | Shared types for Directus                                                                                                        |
+| [@directus/update-check](https://github.com/directus/directus/tree/main/packages/update-check)                           | Check if an update is available for a given package                                                                              |
+| [@directus/utils](https://github.com/directus/directus/tree/main/packages/utils)                                         | Utilities shared between the Directus packages                                                                                   |
 
 ## Tests (`/tests`)
 
-Tests are maintained on a per-package base. This folder contains the platform-wide (end-to-end) tests like [Blackbox](https://github.com/directus/directus/tree/main/tests/blackbox).
+Tests are maintained on a per-package base. This folder contains the platform-wide (end-to-end) tests like
+[Blackbox](https://github.com/directus/directus/tree/main/tests/blackbox).
```

Comment:
I think I'd prefer to link to the "Tests" documentation page instead. Something like:

```suggestion
Tests are maintained on a per-package base. This folder contains the platform-wide (end-to-end) tests. See [Tests](/contributing/tests) for more information.
```